### DISCUSSION TITLE
topic idea

### DISCUSSION THREAD

**Discussion by bakachuuuuuuuuuuujjhjugyi** [Monika-After-Story/MonikaModDev#7852] [issue discussions]
So, I was thinking, Monika likes the player regardless of gender and all so wouldn't that make her pan (sorry if i got the sexuality wrong). I think that could totally be a topic.

**Comment by fromulus** [comment]
While I'm all in for inclusivity and making sure the player themselves get the option to share with Monika the kind of person they are, I really don't think this kind of topic is necessary.

While we can make assumptions on Monika's sexuality, it is at the end of the day just that, assumptions, and I don't think it's so simple to have her affirm that she's one sexuality or another when those types of things can be tough to figure out about yourself and for all we know, different Monikas might have entirely different sexualities due to being in a prolonged relationship with their player who could be any kind of person.

In other words, by making her say "Yeah, I'm Pan/Bi/Heterosexual/Gay/Whatever" we kind of lock every Monika into that sexuality, when it's the kind of thing that can change pretty heavily during one's life and across their experiences.

Especially since, really, what would there even be to say in such a topic, beside her either agreeing or denying the sexuality you're assuming her to be ?
Making her give any kind of opinion on sexual orientations in general is not going to happen since this is a rather political and touchy subject that could accidentally upset someone no matter which side we take (I do not mean to offend anyone by saying that, to be clear). 

At the end of the day, she loves you and only you for the person you are, and that's really the only thing you need to know and that needs to be said.

**Comment by lunulae** [comment]
I'm in agreement that it's something we might want to keep vague in the core mod. I read her as pan, but I have an asexual friend who gets some comfort from the idea that Monika might maybe share this aspect. So I don't really want to hard-canonize any particular interpretation and favor it over another. As From said, the important canon is that she loves *you.* ❤️ 

Too, as much as this is an interest of mine, there's a lot of evolving discussion around bi vs pan labels that I wouldn't want to mischaracterize in a topic about it.

**Comment by username11421** [comment]
Something like this sounds controversial. 

It always seemed that Monika is you-sexual, and I think MAS expresses this really well.
